cohttp-eio/src/io.ml

```diff
@@ -0,0 +1,40 @@
+module IO = struct
+  type 'a t = 'a
+
+  let ( >>= ) v f = f v
+  let return v = v
+
+  type ic = Eio.Buf_read.t
+  type oc = Eio.Buf_write.t
+  type conn = Eio.Net.Sockaddr.stream
```

Comment:
I suggest chaning this to:

`type conn = Eio.Switch.t * Eio.Net.Sockaddr.stream`

The switch can be used by the connection handler to allocate any resources needed (e.g. a fiber).